tests/cpu/Platform_tests.cpp

```diff
@@ -154,7 +154,17 @@ OCIO_ADD_TEST(Platform, create_temp_filename)
     std::set<std::string> uids;
     for (size_t idx=0; idx<TestMax; ++idx)
     {
-        uids.insert(OCIO::Platform::CreateTempFilename(""));
+        struct FGuard
+        {
+            ~FGuard()
+            {
+                std::remove(m_filename.c_str());
+            }
+            
+            std::string m_filename;
+        } file{ OCIO::Platform::CreateTempFile("") };
```

Comment:
If we were to have two functions `CreateTempFile` and `CreateTempFileName` we could just use the latter and remove the guard I guess.